layers/meta-resin-raspberrypi/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,6 @@
+UBOOT_KCONFIG_SUPPORT = "1"
```

Comment:
Nit: Usually I use inherit + variable so you can force with the variable whatever you get from the class. In this case, it's fine because the class uses weak assignments.